cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
@Scotchester It is intended to refer to the playground, yes, but the platform team hasn't decided if build is the definitive place for that, or if we want to standup a separate server. See GHE/CFGOV/platform/issues/164.